src/AppInstallerCLITests/TestHooks.h

```diff
@@ -228,4 +234,19 @@ namespace TestHook
     private:
         std::vector<AppInstaller::Repository::ExtractedIconInfo> m_extractedIcons;
     };
+
+    struct SetExportedManifestTypeResult_Override
+    {
+        SetExportedManifestTypeResult_Override(AppInstaller::Manifest::ManifestTypeEnum manifestType) : m_manifestType(manifestType)

```

Comment:
I don't feel this test override is needed. Just recreate the manifest and turn off schema validation, merged could be read correctly.

With this you were limiting the test to singleton only, while most of the manifests are not singleton

Reply:
Removed test cook and turned off validation